rpc/server.go

```diff
@@ -1,3 +1,6 @@
+// Package rpc provides the HTTP server for for incoming JSON-RPC and websocket requests
+// to swapd from the local host. It is also providing stuff related to the libp2p
+// interactions. ... and types for both ... Need help describing the common theme here.
```

Comment:
Need help with this one.

Reply:
Package rpc provides the HTTP server for for incoming JSON-RPC and websocket requests to swapd from the local host. The answers to these queries come from 3 subsystems: net, personal and swap.






 It is also providing stuff related to the libp2p
interactions. ... and types for both ... Need help describing the common theme here.